extensions/GitExtension/FileExplorerGitIntegration.UnitTest/SandboxHelper.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) Microsoft Corporation.
```

Comment:
I am not sure if it is just me, but I see some unknown characters before the forward slash.

Reply:
What's weirder is that the BOM is not being rendered if you view the file from the **Conversations** tab, but it is rendered in the **Files Changed** tab. Looks like a GitHub bug to me.

Reply:
I started a discussion about this on GitHub community, as I'm not aware of a better way to file a direct bug:
https://github.com/orgs/community/discussions/137771

Reply:
Upvoted!